### ISSUE TITLE
Nearby mobs are still angered when a non_angerable mob is captured

### ISSUE THREAD

**Issue by object-Object** [MehVahdJukaar/Supplementaries#637]
The tag is only checked when a mob is released, but nearby mobs are also angered when it's captured. Pretty sure there needs to be a NON_ANGERABLE check at this line:
https://github.com/MehVahdJukaar/Supplementaries/blob/67debe40c37b9ebd8ee30f640e5697bfa37b7519/common/src/main/java/net/mehvahdjukaar/supplementaries/common/items/AbstractMobContainerItem.java#L315

**Comment by Xelbayria** [comment]
just checking, is this issue related to Supplementaries? if no, then please close.
is it resolved? if yes, then please close.

**Comment by object-Object** [comment]
Yes, this is a bug in Supplementaries. I don't see anything added to the code since this issue was made which would resolve it, but I'm not able to test it right now.

**Comment by Xelbayria** [comment]
alright. When you can test, please isolate Supplementaries and check if the issue still persists. This way we can find out which mod is the culprit. 

of course, the more info, the better it is.

**Comment by MehVahdJukaar** [comment]
Yeah I'll get to this when I'll have time as it's a minor thing. @Xelbayria you can leave ope
